### DISCUSSION TITLE
Multiple Collections

### DISCUSSION THREAD

**Discussion by kpmgeek** [strawberrymusicplayer/strawberry#1534] [Ideas]
It would be great to support multiple collections. This would be ideal for those with a large audiobook collection who wish to keep their books separate, and not affecting their music browsing or associated smartplaylists.

This also could be useful for making it easier to manage podcasts without implementing a full podcatcher like Clementine did.

**Comment by trincowski** [comment]
That would be very cool. I sure would appreciate the option, as I keep deleting and restoring my collection while I'm organising new albums.

**Comment by jonaski** [comment]
https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
